### PR TITLE
Add video trimmer

### LINKED ISSUE #115: Timeline cropping after capture is made
Awesome tool!

A cool feature would be cropping of the timeline to remove parts of the capture that are not important to the flow you are trying to show in the gif (things like "moving your mouse to a button", "time it takes to get back to the menu bar to click to stop recording").

The use case I'm thinking about is using screen captures for blog posts that explain a new feature of a web app.  See the gif I put in [here](https://www.ansible.com/blog/tower-3-search) for an example.  If I would have had an easy tool for cropping off the "stall" at the end of the loop, it would have improved the quality of the gif.

Thinking from a ux-perspective, it seems it would make sense if the bar giving loop position was able to deal with user interaction to set these "time-line crop" points.  Screen shot shows this idea.

screen shot 2016-12-12 at 4 58 35 pm

### PR DESCRIPTION
**Pull request by esalter-va**
This PR adds the ability to select which portion of the screen capture you want to export, which is useful when you want to get rid of dead space around the actual focus of the screen cap. This should resolve #115.

See it in action here:

kap

Here are the files that were exported by Kap in the above gif:

|Format|
|---|
|[gif](https://drive.google.com/open?id=0ByfdP1kAZa_PbjJsWDJIRHVyejA)|
|[mp4](https://drive.google.com/open?id=0ByfdP1kAZa_Pbl8wVDZOS09xbm8)|
|[webm](https://drive.google.com/open?id=0ByfdP1kAZa_PS0o3ZUZ0SmhTTUk)|
|[apng](https://drive.google.com/open?id=0ByfdP1kAZa_PUGQ1OFBxQ1UtQ00)|

### TODO

- [ ] Replace the trimmer handles with something nicer
    - I'm not a designer, so if someone gave me some SVGs to use that would be fantastic.  Otherwise I'll try to make it look decent on my own.

### CODE REVIEW

File: scripts/download-ffmpeg.js [hunk 1]
```diff
@@ -9,7 +9,7 @@ const ora = require('ora');
 
 let spinner = ora({text: 'Installing 7zip', stream: process.stdout}).start();
 
-const FFMPEG_URL = 'https://evermeet.cx/ffmpeg/ffmpeg-85767-gdec2fa8.7z';
+const FFMPEG_URL = 'https://evermeet.cx/pub/ffmpeg/snapshots/ffmpeg-86910-g797c232.7z';
```
Comment (esalter-va) [inline scripts/download-ffmpeg.js]: This is causing a merge conflict, and before I resolve it I just want to make sure this is okay.  I was getting a 404 on `npm install` before making this change.

Comment (esalter-va) [inline scripts/download-ffmpeg.js]: Looks like this has been fixed on master, so I'll take it out.

Comment (CodeTheory) [inline scripts/download-ffmpeg.js]: Sorry for the late reply, @esalter-va. 

Yeah, this has been fixed on master 👌

**Comment by CodeTheory** [comment]
This looks amazing 😍 
Thanks for this!
We'll review it soon

**Comment by sonaye** [comment]
Yes please!

**Comment by sindresorhus** [comment]
Looks great to me too. Thank you so much for working on this @esalter-va 🙌🎂

**Comment by sindresorhus** [comment]
Regarding:

>  Replace the trimmer handles with something nicer
> I'm not a designer, so if someone gave me some SVGs to use that would be fantastic. Otherwise I'll try to make it look decent on my own.

@CodeTheory and @skllcrn will work on this before the next release.

**Pull request merged**